CHANGELOG.md

```diff
@@ -2,7 +2,15 @@
 
 ## Breaking changes
 
-- No breaking changes yet
+- Move `settings.js` to the `settings` directory and deprecate loading
+  `settings.js` from the root. The deprecation warning means this isn't breaking
+  yet, but **this will break eventually**.
```

Comment:
`settings.json` instead of `settings.js`

Reply:
Fixed.